enterprise/frontend/src/embedding-sdk/CHANGELOG.md

```diff
@@ -1,22 +1,3 @@
-## [0.1.40](https://github.com/metabase/metabase/compare/embedding-sdk-0.1.39...embedding-sdk-0.1.40) (2024-10-17)
```

Comment:
Remove all entries after v0.1.38. See the [PR description](https://github.com/metabase/metabase/pull/48985#issue-2605001075) for more details.